packages/core/src/types.ts

```diff
@@ -7,3 +7,20 @@ export type StoreDeploymentConfig = (
 ) => Promise<string>
 
 export type RelayProposal = (proposalRequest: ProposalRequest) => Promise<void>
+
+export type SphinxLock = {
+  format: string
+  orgId: string
+  projects: {
+    [projectName: string]: {
+      projectId: string
+      projectName: string
```

Comment:
It can be helpful to have the data available in the object itself rather than requiring it to be pulled from the key. I don't think it matters that the data is repeated since this file is autogenerated.